app/views/content/start-teacher-training-this-september.md

```diff
@@ -30,8 +30,8 @@ Our teacher training advisers can help you write a strong application quickly.
 They will help you to:
 
 * create a personal statement that shows off your strengths
-* choose the best people to be your references
-* explain any gaps in your work history
+* submit a quality application, including choosing the right references
```

Comment:
Not sure about the word 'quality' on its own here

Reply:
I'd just take this bullet out - references are important on their own, but if the ITT would rather not draw this out then that's fine. I don't think "quality application" adds anything given the other content.